snap/snapcraft.yaml

```diff
@@ -426,6 +425,27 @@ apps:
       # who is initially created as a role with postgres
       - bin/drop
     plugs: [home, removable-media, network]
+  kuiper:
+    adapter: full
+    command: bin/kuiper-server
+    daemon: simple
+    environment:
+      KuiperBaseKey: $SNAP_DATA/kuiper
+      # KUIPER_DEBUG: "true"
+      KUIPER_CONSOLE_LOG: "true"
+      KUIPER_REST_PORT: 48075
+      EDGEX_SERVER: edgex-app-service-configurable-rules
+      EDGEX_SERVICE_SERVER: http://localhost:48080
+      EDGEX_TOPIC: events
```

Comment:
Also don't see where you set `PublishTopic` on app-service-rules to be `events`. By default it is blank for the old rules engine.

Reply:
FYI, this is the override we have on app-service-rules for Kuiper.       
```
Binding_PublishTopic: events
```